app/src/main/java/com/example/sharingang/NewItemActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.sharingang
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityNewItemBinding
+
+class NewItemActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityNewItemBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_new_item)
+    }
+
+    fun createNewItem(view: View) {
+        val intent = Intent(baseContext, MainActivity::class.java)
+        intent.putExtra(
+            "LAST_DESCRIPTION",
+            binding.description
+        )
+        startActivity(intent)
```

Comment:
NewItemActivity shouldn't start the MainActivity, instead it can call setResult() with the intent and then finish()